airflow/providers/teradata/transfers/teradata_to_teradata.py

```diff
@@ -0,0 +1,91 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Sequence
+
+from airflow.models import BaseOperator
+from airflow.providers.teradata.hooks.teradata import TeradataHook
+
+if TYPE_CHECKING:
+    from airflow.utils.context import Context
+
+
+class TeradataToTeradataOperator(BaseOperator):
+    """
+    Moves data from Teradata source database to Teradata destination database.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:TeradataToTeradataOperator`
+
+    :param teradata_destination_conn_id: destination Teradata connection.
```

Comment:
Lets use conventions we already have
You can see for example `source_aws_conn_id` & `dest_aws_conn_id` in [S3FileTransformOperator](https://github.com/apache/airflow/blob/9cb2052810a8a4b191e77d804fc79927f046c8bb/airflow/providers/amazon/aws/operators/s3.py#L567)

so we need `source_teradata_conn_id` / `dest_teradata_conn_id`

Reply:
@SatishChGit

Reply:
Modified to follow convention.